firmware/application/protocols/encoders.hpp

```diff
@@ -54,6 +54,30 @@ namespace encoders {
 
 	// Warning ! If this is changed, make sure that ENCODER_UM3750 is still valid !
 	constexpr encoder_def_t encoder_defs[ENC_TYPES_COUNT] = {
+		// Test OOK Doorbell
+		{
+			"Doorbel",
+			"01", "01",
+			228, 57,
+			{ "1000", "1110" },
+			24,	"AAAAAAAAAAAAAAAAAAAAAAAA",
+			"",
+			141260, 32, // repeat=32
+			32
+		},
+
+		// Test OOK Garage Door
+		{
+			"OH200DC",
```

Comment:
To consider: "OH200DC" is just the garage opener model. In turn, this garage opener uses the "LINEAR DELTA 3"  receiver module, which is the actual receiver for this 8 bits signal, being sold apart, and I suppose it can be used for this and other garage openers. So I suggest changing the name of this encoder into "LINEARDR3" or "LINEAR8bit" or maybe even just "8bits". In short, a name describing the device owning the FCC, or if it is safe to assume so, just 8bits. Makes sense ?

Reply:
Yes, definitely! The `OH200DC` name is temporary, for a test / debug purpose. If you ask me,`8bits` is generic enough, given the fact we don't event know the decoder in the `Linear Delta 3` receiver (and that this is just a atmega328 with a software decoder)